client/commands.c

```diff
@@ -930,25 +930,17 @@ static int out_sort_callback(const void *l, const void *r)
 	return strcmp(address_l, address_r);
 }
 
-static void *add_block_callback(void *block, void *data)
-{
-	unsigned *i = (unsigned *)data;
-	xdag_add_block((struct xdag_block *)block);
-	if(!(++*i % 10000)) printf("blocks: %u\n", *i);
-	return 0;
-}
-
 int out_balances()
 {
 	char address[33] = {0};
 	struct out_balances_data d;
 	unsigned i = 0;
-
+    g_xdag_state = XDAG_STATE_LOAD;
```

Comment:
Why?

Reply:
Originally, I wanted to optimize the performance. When I loaded the local block, I did not synchronize the data. But there was a problem you said. I removed this code